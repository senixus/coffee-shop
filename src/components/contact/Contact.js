import React from "react";

import Navbar from "../navbar/Navbar";

import "./contact.scss";

const Contact = () => {
  return (
    <main className="contact">
      <Navbar />
      <div className="contact-content">
        <form
          className="contact-form"
          action="https://formspree.io/f/xaylzaba"
          method="POST"
        >
          <div className="contact-form__group">
            <label htmlFor="name">Name : </label>
            <input type="text" name="name" placeholder="Name" required />
          </div>
          <div className="contact-form__group">
            <label htmlFor="email">Email : </label>
            <input type="email" name="email" placeholder="Email" required />
          </div>
          <div className="contact-form__group">
            <label htmlFor="message">Message : </label>
            <textarea name="message" required placeholder="Message" />
          </div>
          <div className="contact-form__group ">
            <button type="submit">Send</button>
          </div>
        </form>
      </div>
    </main>
  );
};

export default Contact;
