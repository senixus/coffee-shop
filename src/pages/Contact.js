import Contact from "../components/contact/Contact";
import ReactHelmet from "../components/helmet/ReactHelmet";

const ContactPage = () => {
  return (
    <div>
      <ReactHelmet title="Contact" />
      <Contact />
    </div>
  );
};

export default ContactPage;
