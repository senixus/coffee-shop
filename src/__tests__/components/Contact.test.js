import { shallow } from "enzyme";
import React from "react";
import Contact from "../../components/contact/Contact";

it("expect to render Contact component ", () => {
  const wrapper = shallow(<Contact />);
  expect(wrapper.length).toBe(1);
  expect(wrapper).toMatchSnapshot();
});
