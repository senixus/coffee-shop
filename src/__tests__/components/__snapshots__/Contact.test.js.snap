// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Contact component  1`] = `
<main
  className="contact"
>
  <Navbar />
  <div
    className="contact-content"
  >
    <form
      action="https://formspree.io/f/xaylzaba"
      className="contact-form"
      method="POST"
    >
      <div
        className="contact-form__group"
      >
        <label
          htmlFor="name"
        >
          Name : 
        </label>
        <input
          name="name"
          placeholder="Name"
          required={true}
          type="text"
        />
      </div>
      <div
        className="contact-form__group"
      >
        <label
          htmlFor="email"
        >
          Email : 
        </label>
        <input
          name="email"
          placeholder="Email"
          required={true}
          type="email"
        />
      </div>
      <div
        className="contact-form__group"
      >
        <label
          htmlFor="message"
        >
          Message : 
        </label>
        <textarea
          name="message"
          placeholder="Message"
          required={true}
        />
      </div>
      <div
        className="contact-form__group "
      >
        <button
          type="submit"
        >
          Send
        </button>
      </div>
    </form>
  </div>
</main>
`;
